src/browser/base/zen-components/ZenWorkspaces.mjs

```diff
@@ -27,6 +27,10 @@ var ZenWorkspaces = new (class extends ZenMultiWindowFeature {
       console.warn('ZenWorkspaces: !!! ZenWorkspaces is disabled in hidden windows !!!');
       return; // We are in a hidden window, don't initialize ZenWorkspaces
     }
+    
+    Services.scriptloader.loadSubScript("chrome://browser/content/zen-components/ZenEmojies.mjs", this);
```

Comment:
Please only load the emojis once we need them and unload when the popup is hidden